### PR TITLE
Close Portland 2023 conference website

### PR DESCRIPTION
**Pull request by plaindocs**
I started futzing with the flags, and then the post.

- [x] needs video link (fixes the broken build)
- [x] post needs a better edit 

We probably want to decouple the video archive from this post. 

----
:books: Documentation preview :books:: https://writethedocs-www--1959.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2023/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+-----------
+
+`Videos of the talks`_ are now available on YouTube. This
+includes all conference talks and Q&A, as well as the lightning talks.
+You can also subscribe to our `YouTube channel`_ so you don’t miss any future content.
+
+Thanks again to all our presenters, moderator, and folks who asked wonderful questions!
+Huge thanks also to `Bart at Work`_ who flew out to handle the A/V production to great effect.
+
+.. _Videos of the talks: TODO
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/thanks-recap.rst]:
```suggestion
.. _Videos of the talks: https://www.youtube.com/playlist?list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2
```

I think we actually have a variable for this?

File: docs/_data/portland-2023-config.yaml [hunk 2]
```diff
@@ -65,8 +65,10 @@ buttons:
   #    link: /schedule
   #   - text: Watch the Live Stream
   #     link: /livestream
-  #   - text: Browse the photos
-  #     link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    - text: Conference recap
```
Comment (plaindocs) [inline docs/_data/portland-2023-config.yaml:68]: Don't wanna make this a see the photos?

**Review by plaindocs — COMMENTED** [review]
LGTM

File: docs/_data/portland-2023-config.yaml (hunk 2 above)
Comment (ericholscher) [reply docs/_data/portland-2023-config.yaml:68]: I think linking to the recap has all the info, which is useful.

**Pull request merged**